_episodes/036-luca-palmieri.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "From Zero to Production"
+date: 2021-09-10T15:30:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e036-luca-palmieri.mp3
+duration: "1:07:42"
+length: "65103280"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+#  From Zero to Production  
+
+Allen Wyma talks with Luke Palmieri, a principal engineer, also the book author of ["Zero To Production in Rust"](https://www.zero2prod.com/index.html)
```

Comment:
Would be good to say _where_ they're a principal engineer. "Principal engineer" on its own doesn't say much.

Reply:
Also, this sentence is missing a full stop at the end.

Reply:
Looking at the timestamps, it seems like this is really an interview _about_ the book. That should be clear in the description here.